crates/config/src/config.rs

```diff
@@ -5,28 +5,38 @@ use serde::{Deserialize, Serialize};
 
 use crate::{bridge::RelayerConfig, btcio::BtcioConfig};
 
+/// Default values for some of the [`ClientConfig`] attributes.
 const DEFAULT_RPC_PORT: u16 = 8542;
 const DEFAULT_P2P_PORT: u16 = 8543;
 const DEFAULT_DATADIR: &str = "strata-data";
```

Comment:
This docstring only applies to `DEFAULT_RPC_PORT`.
Can you please add for the ones?

Reply:
Oops..